Sources/TuistCore/Automation/XcodeBuildControlling.swift

```diff
@@ -12,6 +12,26 @@ public enum XcodeBuildTestAction: Equatable {
     case test
     case build
     case testWithoutBuilding
+
+    public var description: String {
+        switch self {
+        case .test:
+            "Testing"
+        case .build:
+            "Building"
+        case .testWithoutBuilding:
+            "Testing without building"
+        }
+    }
+
+    public var verb: String {
+        switch self {
+        case .test, .testWithoutBuilding:
+            "ran"
+        case .build:
+            "built"
+        }
+    }
```

Comment:
This feels very much tailored to the output in the `TestService`. I'd move the extension there, so we don't unnecessarily increase the scope of the public API of `TuistCore`.

Reply:
Moved this to the place it was used